### PR TITLE
Add bread to food list and add "go through modes of documentation" to suggestions

### PR DESCRIPTION
**Pull request by EthanBirdB**
There wasn't enough bread in that list.

### CODE REVIEW

**Comment by JesseWilliamson** [comment]
What type of bread?

**Review by JesseWilliamson — DISMISSED** [review]

File: Foods.yaml [hunk 1]
```diff
@@ -6,4 +6,4 @@
 
 FavouriteFoods:
   # Add the food to this list on this line, if possible.
-  - ["pizza", "kale_smoothies", "bacon", "cheese", "pie", "carbonara", "garlic_bread", "banana", "burrito", "pasta", "coffee", "chocolate", "chicken_schnitzel", "mango", "macadamias","Tacos","salmon sashimi", "apple", "burger"]
\ No newline at end of file
+  - ["pizza", "kale_smoothies", "bacon", "cheese", "pie", "carbonara", "garlic_bread", "banana", "burrito", "pasta", "coffee", "chocolate", "chicken_schnitzel", "mango", "macadamias","Tacos","salmon sashimi", "apple", "burger", "bread"]
```
Comment (A-Herfel) [inline Foods.yaml]: There is too much bread on this list

**Comment by EthanBirdB** [comment]
> What type of bread?
bread.

**Comment by EthanBirdB** [comment]
> What type of bread?

bread

**Review by PiiDish — DISMISSED** [review]
LGTM

**Comment by ysims** [comment]
Looks like you've got some merge conflicts to fix up

**Comment by ysims** [comment]
Looks good! You just need to fix up your PR title. It should finish the sentence 'when merged, this PR will...'

**Review by ysims — APPROVED** [review]
LGTM (looks good to me)

**Pull request merged**